src/components/cards/style.css

```diff
@@ -0,0 +1,20 @@
+.cardsWrapper {
```

Comment:
Order the properties by alphabetical order

Reply:
Done